nexus_constructor/gnomon.py

```diff
@@ -0,0 +1,326 @@
+from PySide2.Qt3DCore import Qt3DCore
+from PySide2.Qt3DExtras import Qt3DExtras
+from PySide2.Qt3DRender import Qt3DRender
+from PySide2.QtGui import QVector3D, QMatrix4x4, QColor, QFont, QVector4D
+
+
+class Gnomon:
+    def __init__(self, root_entity, main_camera, component_adder):
+        """
+        A class that houses the Qt3D items (entities, transformations, etc) related to the gnomon.
```

Comment:
people new to this project aren't necessarily going to know what a gnomon is. I think we should add something here or where this is constructed to say it's used for the axis indicator.